automation/taskcluster/l10n/locales.py

```diff
@@ -4,131 +4,31 @@
-#
-# Other builds might include more locales.
+
+# In Focus, this file contained a simple list of release locales. Fenix documents this list in
+# '/l10n.toml', so this file instead converts its values to be easily consumed from Python.
 #
-# Note that there are differences in the locale codes used by Pontoon
-# and by Android (e.g. Hebrew: he (Pontoon) vs. iw (Android)).
-# This list uses the Android notation. A valid entry can be a
-# language code (de) or a language plus country (de-DE). Do not use
-# the name of the Androidresource folder (de-rDE) here.
+# Alternatives to custom parsing that were considered:
+# - Using standard library module --- none exists to parse TOML
+# - Importing a TOML parsing module --- introduces an additional build step, complexity, and
+#   security risk
+# - Vendoring a TOML module --- large amount of code given the use case. Introduces a security
+#   risk
 #
-# Releases should contain all locales that are at 100% or have been
-# shipping in previous releases. Ping :delphine in case you want
-# to add or remove locales from releases.
-RELEASE_LOCALES = [
-    "af",
-    "am",
-    "an",
-    "anp",
-    "ar",
-    "ast",
-    "ay",
-    "az",
-    "bg",
-    "bn-BD",
-    "bn-IN",
-    "bo",
-    "bs",
-    "ca",
-    "cak",
-    "co",
-    "cs",
-    "cy",
-    "da",
-    "de",
-    "dsb",
-    "el",
-    "en-CA",
-    "eo",
-    "es-AR",
-    "es-CL",
-    "es-ES",
-    "es-MX",
-    "eu",
-    "fa",
-    "fi",
-    "fr",
-    "fy-NL",
-    "ga-IE",
-    "gu-IN",
-    "gl",
-    "hi-IN",
-    "hr",
-    "hsb",
-    "hu",
-    "hus",
-    "hy-AM",
-    "ia",
-    "in",
-    "it",
-    "ixl",
-    "iw",
-    "ja",
-    "ka",
-    "kab",
-    "kk",
-    "ko",
-    "kw",
-    "lo",
-    "lt",
-    "meh",
-    "mix",
-    "mr",
-    "ms",
-    "my",
-    "nb-NO",
-    "ne-NP",
-    "nl",
-    "nn-NO",
-    "oc",
-    "pai",
-    "pa-IN",
-    "pl",
-    "ppl",
-    "pt-BR",
-    "quc",
-    "quy",
-    "ro",
-    "ru",
-    "sk",
-    "sl",
-    "sn",
-    "sq",
-    "sr",
-    "sv-SE",
-    "su",
-    "ta",
-    "te",
-    "th",
-    "tr",
-    "trs",
-    "tsz",
-    "tt",
-    "uk",
-    "ur",
-    "vi",
-    "wo",
-    "yua",
-    "zam",
-    "zh-CN",
-    "zh-TW"
-]
+# 'l10n.toml' has a very simple structure, and it is reasonable to believe that this (very basic)
+# algorithm will continue to work as it is changed.
+def get_release_locales():
+    with open(r"l10n.toml") as f:
+        file = f.read()
+
+        locales_begin = file.find("locales = [") + len("locales = [")
+        locales_end = file.find("]", locales_begin)
+
+        locales = file[locales_begin:locales_end]
 
-# This is the list of locales that we want to take automated screenshots of
-# in addition to the list of release locales. We want to take screenshots
-# of other locales so that translators of not yet completed locales can
-# verify their work in progress.
+        return locales.split(",")
```

Comment:
repl'ing locally (granted, it's python3), this returns entries like, `'\n  "ca"'`: don't we need to trim out the whitespace and the quotes inside the string?

tbh, doing it this way (with `find` and `split`) makes it difficult to guarantee the formatting hasn't changed. If the formatting changes, we should throw because it might ruin our ability to parse it.

To guarantee formatting, it might be a little simpler to do line by line (though it's non-declarative and there may be a better declarative way to do it). Pseudo-code:
```
for line in lines:
  if line == 'locales = [':
    found = true
    in_locales = true

  if in_locales == true:
    if line == ']':
      break
    else
      locales.append(getLocaleFromStringLine(line)) # and throw if format is not like `   "en-US",`

if found == false: throw
```